<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsUi</class>
 <widget class="QDialog" name="SettingsUi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>261</width>
    <height>320</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label_12">
         <property name="text">
          <string>Audio Codec:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="audioCodecBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_11">
         <property name="text">
          <string>Audio Quality:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="audioQualitySlider">
         <property name="maximum">
          <number>4</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="text">
          <string>Audio Sample Rate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="audioSampleRateBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Container Format:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="containerFormatBox"/>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Video Resolution:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="videoResolutionBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_9">
         <property name="text">
          <string>Video Framerate:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="videoFramerateBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QLabel" name="label_6">
         <property name="text">
          <string>Video Codec:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QComboBox" name="videoCodecBox"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_8">
       <item>
        <widget class="QLabel" name="label_7">
         <property name="text">
          <string>Video Quality:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSlider" name="videoQualitySlider">
         <property name="maximum">
          <number>4</number>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
